service/handlers.go

```diff
@@ -433,11 +417,17 @@ func (h handler) getWorkflowLogs(w http.ResponseWriter, r *http.Request) {
 
 	l := h.requestLogger(r, "op", "get-workflow-logs", "workflow", workflowName)
 
+	if err := validations.ValidateVar("workflow name", workflowName, "is_alphanumunderscore"); err != nil {
```

Comment:
does this come from argo workflows? what happens if they change the format (add or remove a char as being valid)?

Reply:
Yeah it does, we had a test for this but we weren't validating it anywhere so I added this. Maybe remove this and the test then?

Reply:
yeah, i think dropping it makes sense. we're just a pass through for that data at this point